Cargo.toml

```diff
@@ -100,7 +96,6 @@ thrift = "0.17.0"
 tokio = { version = "1.36", default-features = false }
 typed-builder = "0.20"
 url = "2.5.4"
-urlencoding = "2"
```

Comment:
Cargo removed this line and the other non-`paste` lines when dropping `paste` via `cargo remove paste -p iceberg`. I'm happy to add them back if Cargo was over-eager.